wordpress-mobile/WordPress-Android
# Post upload: Revert to stock ProgressBar drawables

As an alternative to #6290, this one uses stock Android drawables for the ProgressBar so we make sure it works alright on all devices.

This is a rather conservative approach that I'd prefer to implement, we might in the future get a revision from design where we get the right assets to build our own ProgressBar

This is how the indeterminate progress bar would look / looks (forced by code to make the pic):
![infiniteprogress](https://user-images.githubusercontent.com/6597771/28196006-46fbd6e4-6824-11e7-986f-4161ef4e268e.gif)

This is how a determinate progress bar would look, at 25% (forced by code as well):
<img width="694" alt="screen shot 2017-07-13 at 11 21 26 pm" src="https://user-images.githubusercontent.com/6597771/28196030-6b098932-6824-11e7-981b-f575e595bacd.png">

And this is how the implementation in this PR looks for an actual Post upload:
<img width="694" alt="screen shot 2017-07-13 at 11 32 27 pm" src="https://user-images.githubusercontent.com/6597771/28196012-4dbaf5b4-6824-11e7-9653-553d1fcbe453.png">


To test:
1. go to the Posts list and start a new draft
2. include some media
3. exit the editor
4. the newly created Post should show the indeterminate progress bar while it uploads related media and the post itself, until it's fully uploaded and then such progress bar disappears.

Test it on pre and post-Lollipop devices.

cc @nbradbury
